things/postgres/things.go

```diff
@@ -171,22 +171,33 @@ func (tr thingRepository) RetrieveByKey(_ context.Context, key string) (string,
 	return id, nil
 }
 
-func (tr thingRepository) RetrieveAll(_ context.Context, owner string, offset, limit uint64, name string) (things.ThingsPage, error) {
+func (tr thingRepository) RetrieveAll(_ context.Context, owner string, offset, limit uint64, name string, metadata things.ThingMetadata) (things.ThingsPage, error) {
 	name = strings.ToLower(name)
 	nq := ""
+	var m []byte
```

Comment:
Why is this declared here? I see it inicialilzed below, where is can be auto-declared.

Reply:
BTW. Go's reccomended/idiomatic way of declaration is using `:=` in favor of `var`, so lean to this approach whenever possible.

Reply:
Following FP principles use var/const when you are declaring empty values.